platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/GradlePluginDevelopmentExtension.java

```diff
@@ -91,8 +81,6 @@ public GradlePluginDevelopmentExtension(Project project, SourceSet pluginSourceS
         this.plugins = project.container(PluginDeclaration.class);
```

Comment:
❓ Don't we need to remove also this one? 🤔

Reply:
yes, that's what I get for trying to resolve the merge on the command-line...